### ISSUE TITLE
Remove usage of `baseInputs` to avoid confusion?

### ISSUE THREAD

**Issue by martin-braun** [NixOS/nix-pills#219]
I'm coming from #87. Reading through Nix Pills, [I can still find numerous references to `baseInputs`](https://www.google.com/search?q=inurl%3Anixos.org%20%22baseInputs%22) that caused great confusion on my end as a newcomer when transitioning to `mkDerivation`, please take the time to get rid of things [that aren't used in the `mkDerivation` at all](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20baseInputs&type=code), because you end up in false assumptions from the things you learned previously. 

Unlearning is significantly harder than learning.

**Comment by MikHulk** [comment]
Yes the sentence suggests it is an alternative but the code without this change  doesn't work. It would be much better to insist on alternatives: merge buildInputs vs keep them in two attributes.
